### PR TITLE
docs: landing page

### PR DESCRIPTION
**Pull request by InnaAtanasova**
This is the implementation of the landing page. There are a few minor things to be changed when Leo gets back from vacation (update 1-2 logos and the background images in the first section and "More About Fundamental Library Setion). Here are the screenshots from Zeplin (the visuals):

Desktop:
Screen Shot 2020-01-03 at 1 56 12 PM
Screen Shot 2020-01-03 at 1 57 39 PM
Screen Shot 2020-01-03 at 1 57 34 PM
Screen Shot 2020-01-03 at 1 57 21 PM

Mobile:
Screen Shot 2020-01-03 at 1 56 50 PM
Screen Shot 2020-01-03 at 1 56 43 PM
Screen Shot 2020-01-03 at 1 56 37 PM
Screen Shot 2020-01-03 at 1 56 28 PM

### CODE REVIEW

**Comment by estelaxu3** [comment]
It looks awesome, thank you for bring it to life!

Couple things I noticed:
1. Key feature content should be left aligned instead of centered. 
Zeplin file:
Screen Shot 2020-01-03 at 2 13 04 PM

2. Could we add hover effect to the library tiles? All four tiles are currently outlined and have shadows, maybe we can take out the shadow when the tile is not selected so it is more interactive and responsive.

**Comment by Vanessa-Cusmich** [comment]
Agreed, it looks great!

I was also thinking we should remove the links below the social media logos and hyperlink the images.

Thanks!
Vanessa

**Comment by InnaAtanasova** [comment]
> It looks awesome, thank you for bringing it to life!
> 
> Couple things I noticed:
> 
> 1. Key feature content should be left aligned instead of centered.
>    Zeplin file:
>    Screen Shot 2020-01-03 at 2 13 04 PM
> 2. Could we add hover effect to the library tiles? All four tiles are currently outlined and have shadows, maybe we can take out the shadow when the tile is not selected so it is more interactive and responsive.

Done

**Comment by InnaAtanasova** [comment]
> Agreed, it looks great! I was also thinking we should remove the links below the social media logos and hyperlink the images. Thanks! Vanessa >.

Done

**Comment by bcullman** [comment]
pls. recheck layout on mobile - footer text is overlapping.

**Comment by InnaAtanasova** [comment]
> pls. recheck layout on mobile - footer text is overlapping.

It's fixed now. Was a Safari issue. Thanks for catching it

File: docs/_sass/_landingPage.scss [hunk 1]
```diff
@@ -0,0 +1,721 @@
...
+
+@media (max-width: 1024px){
+    .features-section {
+        &__container {
+            grid-template-columns: auto auto;
+        }
+    }
+}
+
+@media (max-width: 768px){
```
Comment (JKMarkowski) [inline docs/_sass/_landingPage.scss]: What about adding only one media query per `max-width`?
From largest (1024px) to smallest (600px)

Comment (JKMarkowski) [reply docs/_sass/_landingPage.scss]: The rest is perfect.

Comment (InnaAtanasova) [reply docs/_sass/_landingPage.scss]: Doesn't look good without this breakpoint. O.w. it's not a problem to change it from 768 to 600px

Comment (InnaAtanasova) [reply docs/_sass/_landingPage.scss]: This is how the section looks like at 630px:
Screen Shot 2020-01-07 at 8 17 48 AM

This is how the section looks like compared to the other sections:
Screen Shot 2020-01-07 at 8 18 12 AM
Screen Shot 2020-01-07 at 8 18 32 AM

**Comment by estelaxu3** [comment]
LGTM

**Pull request merged**